src/Components/test/E2ETest/Tests/BootResourceCachingTest.cs

```diff
@@ -148,8 +148,8 @@ private IReadOnlyCollection<string> GetAndClearRequestedPaths()
 
         private void WaitUntilLoaded()
         {
-            new WebDriverWait(Browser, TimeSpan.FromSeconds(30)).Until(
-                driver => driver.FindElement(By.TagName("h1")).Text == "Hello, world!");
+            var element = Browser.Exists(By.TagName("h1"));
```

Comment:
Why do we need to change this?

Reply:
This test failed when I was running locally. Using `Browser.Exists` gets us the build logs which this one does not. In my case it was because something about the build had gone wrong and it failed to download a bunch of files which is the next mystery to solve.